src/include/mit/mit.h.in

```diff
@@ -368,35 +368,39 @@ static int store_stack(mit_word * restrict stack, mit_uword stack_depth,
 
 // VM instantiation and control
 
-mit_state *mit_init(size_t memory_size, size_t stack_size);
+mit_state *mit_init(size_t memory_bytes, size_t stack_words);
 /* Create and initialize a virtual machine state.
 
-   - memory_size - number of words of memory required
-   - stack_size - number of words of stack space required
+   - memory_bytes - number of bytes of memory required
+   - stack_words - number of words of stack space required
 
-   Return a pointer to a new state, or `NULL` if memory cannot be
-   allocated, or if any requested size is larger than `MIT_UWORD_MAX`.
+   The sizes must be multiples of `WORD_BYTES`.
```

Comment:
I suggest "must be a whole number of words". It's confusing to say `stack_words` must be a multiple of `word_bytes`.

Reply:
Good catch!